src/components/UploadPhoto/UploadPhoto.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { ChangeEvent } from 'react';
+import defaultPhoto from '../../assets/defaultPhoto.jpg';
+import './UploadPhoto.css';
+
+interface UploadPhotoProps {
```

Comment:
Maybe put the interface in the types file?

Reply:
see previous comment